README.md

```diff
@@ -232,6 +232,12 @@ Global hook which will run immediately before send API response.
 
 Global hook which will run immediately after sent API response back to client.
 
+##### log\_error\_on\_finish (global)
```

Comment:
Shouldn't this be `on_error` instead?

Reply:
oops mybad, forgot to update this to latest change. Have just updated them accordingly. Thanks for spotting !